src/databricks/labs/ucx/assessment/queries/all_tables.sql

```diff
@@ -0,0 +1,14 @@
+-- viz type=table, name=Table Types, columns=database,name,type,table_format,table_view,storage,is_delta,location
+-- widget title=Table Types, col=0, row=0, size_x=6, size_y=6
+SELECT
+  `database`,
+  name,
+  object_type AS type,
+  upper(table_format) AS format,
+  CASE WHEN object_type IN ("MANAGED", "EXTERNAL") THEN "TABLE" ELSE "VIEW" END AS table_view,
+  CASE WHEN SUBSTRING(location,1,4)="dbfs" AND SUBSTRING(location,6,4)<>"/mnt" THEN "DBFS ROOT"
```

Comment:
Can you replace substring() everywhere with a more readable startswith()? The reason is I don't remember if it starts with 0 or 1.

Internally it also is going through a faster execution path 

https://docs.databricks.com/en/sql/language-manual/functions/startswith.html

Reply:
@FastLee is this checking for `dbfs://mnt` or `/dbfs/mnt` - looks like the latter. Why not check for `startswith('/dbfs/mnt/)`?

Reply:
Ah, this is then
```
STARTSWITH(location, "/dbfs") AND NOT STARTSWITH(location, "/dbfs/mnt")
```